src/js/tech/html5.js

```diff
@@ -321,7 +321,7 @@ class Html5 extends Tech {
     if ('webkitDisplayingFullscreen' in video) {
       this.one('webkitbeginfullscreen', function() {
         this.one('webkitendfullscreen', function() {
-          this.trigger('fullscreenchange');
+          this.trigger('fullscreenchange', { isFullscreen: false });
         });
 
         this.trigger('fullscreenchange');
```

Comment:
Probably because the player was still getting the `fullscreenchange` and up until this point everything is similar to the normal, expected, spec behavior. Just that exiting the fullscreen doesn't get set correctly.